docs/json/radarr/cf/german-lq-release-title.json

```
@@ -23,6 +23,15 @@
       "fields": {
         "value": "Jellyfin-Plex$"
       }
+    },
+    {
+      "name": "Double German Tag",
+      "implementation": "ReleaseTitleSpecification",
+      "negate": false,
+      "required": false,
+      "fields": {
+        "value": "\\.German\\..+?\\.?German$"
```

```suggestion
        "value": "[._-]German[._-].+?[._-]?German$"
```
this would cover additional cases for different delimiters, might even want to add a whitespace in there if that's common in german releases